src/v2/Components/NavBar/Menus/UserMenu.tsx

```diff
@@ -39,39 +42,59 @@ export const UserMenu: React.FC = () => {
   const hasPartnerAccess = Boolean(user.has_partner_access)
 
   return (
-    <Menu onClick={trackClick}>
-      {isAdmin && <MenuItem href={sd.ADMIN_URL}>Admin</MenuItem>}
-      {(isAdmin || hasPartnerAccess) && (
-        <MenuItem href={sd.CMS_URL}>CMS</MenuItem>
+    <Menu>
+      {isAdmin && (
+        <MenuItem href={sd.ADMIN_URL} onClick={trackClick}>
+          Admin
+        </MenuItem>
       )}
+
       {(isAdmin || hasPartnerAccess) && (
-        <Flex width="100%" justifyContent="center" my={1}>
-          <Box width="90%">
-            <Separator />
-          </Box>
-        </Flex>
+        <>
+          <MenuItem href={sd.CMS_URL} onClick={trackClick}>
+            CMS
+          </MenuItem>
+
+          <Flex width="100%" justifyContent="center" my={1}>
+            <Box width="90%">
+              <Separator />
+            </Box>
+          </Flex>
+        </>
       )}
+
       {isAdmin && (
-        <MenuItem href="/user/purchases">
-          <TagIcon mr={1} /> Purchases
+        <MenuItem href="/user/purchases" onClick={trackClick}>
+          <TagIcon mr={1} title="View your purchases" /> Purchases
         </MenuItem>
       )}
-      <MenuItem href="/user/saves">
-        <HeartIcon mr={1} /> Saves & Follows
+
+      <MenuItem href="/user/saves" onClick={trackClick}>
+        <HeartIcon mr={1} title="View your Saves &amp; Follows" /> Saves &amp;
+        Follows
       </MenuItem>
-      <MenuItem href="/profile/edit">
-        <SoloIcon mr={1} /> Collector Profile
+
+      <MenuItem href="/profile/edit" onClick={trackClick}>
+        <SoloIcon mr={1} title="View your Collector Profile" /> Collector
+        Profile
       </MenuItem>
-      <MenuItem href="/user/edit">
-        <SettingsIcon mr={1} /> Settings
+
+      <MenuItem href="/user/edit" onClick={trackClick}>
+        <SettingsIcon mr={1} title="Edit your settings" /> Settings
       </MenuItem>
+
       <MenuItem
-        onClick={event => {
-          event.preventDefault() // `href` is only for tracking purposes
+        tabIndex={0}
+        onKeyPress={event => {
+          if (event.key === "Enter" || event.key === " ") {
```

Comment:
I was *just* looking at this because one of the specs picks up that the situation is a bit more annoying: the notifications bell is ALSO a link to the works-for-you page. So we actually have to support both the click for the anchor and the keypress for the subnav expansion.

But to your point:
https://www.w3.org/WAI/GL/wiki/Making_actions_keyboard_accessible_by_using_keyboard_event_handlers_with_WAI-ARIA_controls#Example_1:_Using_Space_to_activate_a_button

> Note that supporting SPACE in addition to ENTER is not required by WCAG 2.0; this control would still be keyboard accessible without the key event handler for SPACE. However, supporting platform keyboard conventions for controls is strongly encouraged.